### ISSUE TITLE
RFE - Add New Accepted Ignores to the `sanity` test

### ISSUE THREAD

**Issue by alisonlhart** [ansible/ansible-lint#3694]
Add new allowed ignores to the ansible-lint `sanity` ignore file. This is to allow Certified and Validated content to ignore certain tests due to known false positives, or if the ignores are not for role or plugin content. 

The test should be reworked to only check for ignores for files under `plugins` or `roles` directories. 

The following ignores should be added to the allowed list: 

```
- shellcheck
- shebang
```

(Note: PE will take ownership for adding this)

**Comment by alisonlhart** [comment]
This is completed
